orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -334,17 +343,13 @@ def forward(self, *inputs, **kwargs):
                     raise RuntimeError('A device must be specified in the model or data!')
             self._get_inference_graph_and_init_gradient_graph_builder(*inputs, **kwargs)
 
-        _, _, input_names_require_grad = _parse_inputs_for_onnx_export(self._original_module, *inputs, **kwargs)
+        _, _, input_names_require_grad, new_input_shape = _parse_inputs_for_onnx_export(self._original_module_input_names, self._onnx_inference, *inputs, **kwargs)
```

Comment:
`_parse_inputs_for_onnx_export` is a `ORTModule` method, so you don't need to pass  `self._original_module_input_names` and `self._onnx_inference` as external arguments. It can access them directly and simplify the method signature

Reply:
`_parse_inputs_for_onnx_export` is a standalone function, not a method.